docs/config/lua/keyassignment/ScrollByPage.md

```diff
@@ -2,6 +2,7 @@
 
 Adjusts the scroll position by the number of pages specified by the argument.
 Negative values scroll upwards, while positive values scroll downwards.
+Float values will scroll by partial pages (e.g. 0.5 will scroll half a page down).
```

Comment:
Thanks for adding some docs!  What I like to do for these is make a section like this:

```
*Since: nightly builds only*

You may now use floating point values to scroll by partial pages.  This example shows
how to make the `PageUp`/`PageDown` scroll by half a page at a time:

<insert example here>
```

I would put that whole block below the existing example